YotiButtonSDK/YotiSDK.swift

```diff
@@ -13,6 +13,9 @@ import UIKit
 public class YotiSDK: NSObject {
 
     static let shared = YotiSDK()
+    @objc public static let didFinishNetworkRequest = Notification.Name("yoti.didFinishNetworkRequest")
+    @objc public static let willMakeNetorkRequest = Notification.Name("yoti.willMakeNetorkRequest")
```

Comment:
Prefer to move those into a separated level for clearness, e.g.
`@objc(YTBNotification) struct YotiSDK.Notification`

and recommended using the standard reverse-DNS naming convention
(e.g. `com.yoti.mobile.ios.sdk.YotiButton.notificaiton.willMakeNetworkRequest`)